cmd/bbolt/main.go

```diff
@@ -1604,6 +1611,23 @@ func (cmd *BenchCommand) runReadsSequentialNested(db *bolt.DB, options *BenchOpt
 	})
 }
 
+func (cmd *BenchCommand) checkProgress(results *BenchResults, finishChan chan interface{}) {
+	ticker := time.Tick(time.Second)
+	lastCompleted, lastTime := int64(0), time.Now()
+	for {
+		select {
+		case <-finishChan:
+			return
+		case t := <-ticker:
+			completed, taken := results.CompletedOps, t.Sub(lastTime)
+			fmt.Fprintf(cmd.Stderr, "Completed %d requests, %3.1f/s \n",
+				completed, float64(completed-lastCompleted)*float64(int64(time.Second))/float64(int64(taken)),
```

Comment:
does the double cost (int -> float) helps here ?

Reply:
removed unnecessary conversion. but 'completed'  is int64 type, int support only 2 billions writes.